lib/metadataTypes/EmailSendDefinition.js

```diff
@@ -147,7 +144,7 @@ class EmailSendDefinition extends MetadataType {
             // get List (required)
             if (sdl.r__list_PathName) {
                 sdl.List = {
-                    ID: Util.getListObjectIdFromCache(this.cache, sdl.r__list_PathName, 'ID'),
+                    ID: cache.getListObjectIdFromCache(sdl.r__list_PathName, 'ID'),
```

Comment:
this method wasn't moved to Cache class but has or else this fails

- [ ] move getListObjectIdFromCache from util.js to cache.js